core/templates/pages/about-page/about-page.component.html

```diff
@@ -43,20 +43,341 @@ <h4 class="oppia-about-feature-text" [innerHTML]="feature.i18nDescription | tran
+                  <td style="--size: 0.39;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_KENYA' | translate }} </span></th>
+                  <td style="--size: 0.27;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_CAMEROON' | translate }} </span></th>
+                  <td style="--size: 0.18;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_MOZAMBIQUE' | translate }} </span></th>
+                  <td style="--size: 0.1;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_INDIA' | translate }} </span></th>
+                  <td style="--size: 0.1;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_NIGERIA' | translate }} </span></th>
+                  <td style="--size: 0.09;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_INDONESIA' | translate }} </span></th>
+                  <td style="--size: 0.09;"><span class="data"></span></td>
+                </tr>
+                <tr>
+                  <th scope="row"><span> {{'I18N_ABOUT_PAGE_COUNTRY_TANZANIA' | translate }} </span></th>
+                  <td style="--size: 0.08;"><span class="data"></span></td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-about-impact-section-buttons">
+      <div class="oppia-about-impact-section-button-container">
+        <p class="oppia-about-section-subtext-new oppia-about-impact-buttons-content">
+          {{ 'I18N_ABOUT_PAGE_IMPACT_SECTION_REPORT_BUTTON_SUBTEXT' | translate }}
+        </p>
+        <div>
+          <oppia-primary-button
+            buttonHref="https://drive.google.com/file/d/1uRe145ou9Ka5O2duTB-N-i89NVPEtxh1/view"
+            [customClasses]="['oppia-about-impact-button']"
+            buttonText="{{ 'I18N_ABOUT_PAGE_IMPACT_SECTION_REPORT_BUTTON_TEXT' | translate }}">
+          </oppia-primary-button>
+        </div>
+      </div>
+      <div class="oppia-about-impact-section-button-container">
+        <p class="oppia-about-section-subtext-new oppia-about-impact-buttons-content">
+          {{ 'I18N_ABOUT_PAGE_IMPACT_SECTION_UPDATES_BUTTON_SUBTEXT' | translate }}
+        </p>
+        <div>
+          <oppia-primary-button
+            buttonHref="https://lookerstudio.google.com/u/0/reporting/dbc21029-2acd-44e7-aa46-504647c295fb/page/p_rq5cam1g8c"
```

Comment:
This links require permissions to view, not sure why are we linking it in a public page.